.changeset/poor-loops-boil.md

```diff
@@ -0,0 +1,24 @@
+---
+'astro': minor
+---
+
+Adds the `ActionInputSchema` utility type to automatically determine the TypeScript type of an action's input based on its Zod schema
```

Comment:
```suggestion
Adds the `ActionInputSchema` utility type to automatically infer the TypeScript type of an action's input based on its Zod schema
```
We did this in docs, in case you prefer it, and for consistency!